src/tests/functional/visit_tests.ts

```diff
@@ -71,6 +71,14 @@ export class VisitTests extends TurboDriveTestCase {
     this.resetRequestInterceptor()
   }
 
+  async "test turbo:before-fetch-request event.detail"() {
```

Comment:
I am getting a timeout error when I run the whole suite, but everything seems to work if I run this specific test in isolation (by adding `intern.configure({ grep: /turbo:before/ })` in `runner.js`).

It's strange since it's basically identical to the request interceptor test.


```
× firefox on linux 5.10.0-0.bpo.5-amd64 - VisitTests - turbo:before-fetch-request event.detail (2.5s)
    TimeoutError: Timeout reached on firefox on linux 5.10.0-0.bpo.5-amd64 - VisitTests - turbo:before-fetch-request event.detail#
      at Timeout._onTimeout @ src/lib/Test.ts:226:24
      at ontimeout @ timers.js:436:11
      at tryOnTimeout @ timers.js:300:5
      at listOnTimeout @ timers.js:263:5
      at Timer.processTimers @ timers.js:223:10


```

Reply:
We are seeing the same on #299. Only for Firefox. Odd. Please do investigate!